proj/hps_accel/src/blocks.cc

```diff
@@ -77,10 +77,7 @@ Vector16 Vector16::build(int8_t a, int8_t b, int8_t c, int8_t d, int8_t e,
 
 Vector16 Vector16::zeroes() { return Vector16{{0, 0, 0, 0}}; }
 
-// Performs a 16 x 16 vector multiplication
-int32_t multiply_accumulate() {
-  return cfu_get(REG_MACC_OUT);
-}
+extern int32_t multiply_accumulate();
```

Comment:
Why is this needed? Are there link errors without it?

Reply:
It ensures there is a symbol defined for the inline function. I originally saw this rule in the Opentitan style guide:

https://docs.opentitan.org/doc/rm/c_cpp_coding_style/#:~:text=compilation%20unit%20with%20a%20compatible%20extern%20declaration

and I assumed there was a similar rule in the Google C++ style guide, although I see now there isn't.

We don't actually need these. Although these inline functions *are* exported, they are only exported to the other files in our project here, so if we did need a real symbol defined for them we would know about it.

I'll remove them.

Reply:
Oh, I see it matches C11. That's a good thing to keep in mind.